lang/fr.json

```
@@ -1,68 +1,68 @@
-  "common.plural_rules.other": "th",
+  "common.plural_rules.one": "er",
+  "common.plural_rules.two": "ème",
+  "common.plural_rules.few": "ème",
+  "common.plural_rules.other": "ème",
   "header.credits": "Illustrations de <2>Kate Rodman</2>",
   "navigation.name": "Menu",
   "navigation.close": "Fermer le menu",
   "navigation.made_with_love": "Made with love by the <2>Vegan Hacktivists</2>.",
   "introduction.id": "introduction",
   "introduction.name": "Introduction",
-  "introduction.tagline": "...et pourquoi c'est important",
+  "introduction.tagline": "... et pourquoi c'est important",
   "introduction.copy": [
     "Lorsque nous pensons à la nature et à la vie des animaux sauvages, nous imaginons souvent des dauphins nageant dans l'océan ou des éléphants et des lions errant dans la savane. Nous imaginons surtout de grands animaux adultes menant une vie caractérisée par la liberté. Cependant, cette vision teintée de rose de la nature est loin de représenter la vie de la plupart des animaux sauvages.",
     "En réalité, la vie de la plupart des animaux sauvages n'est pas caractérisée par la liberté, mais par la lutte. Ils sont constamment confrontés à des conditions climatiques difficiles, à la famine, aux maladies et aux blessures, et ils souffrent de la même manière qu'un chien ou un être humain."
   ],
   "introduction.overview": "Voici un aperçu de la souffrance des animaux sauvages, de son importance et de ce que nous pouvons faire pour les aider.",
-  "scale_of_suffering.id": "scale-of-suffering",
+  "scale_of_suffering.id": "l-echelle-de-la-souffrance",
   "scale_of_suffering.name": "L'échelle de la souffrance",
   "scale_of_suffering.humans": "<0>Il y a environ</0><1>8 milliards d'êtres humains</1><2> sur la planète Terre.</2>",
   "scale_of_suffering.farmed_animals": "<0>Pour chaque être humain, il y a environ</0> <1>3-4 animaux d'élevage</1> <2>qui sont élevés à un moment donné.",
-  "scale_of_suffering.farmed_fish": "...sans compter les poissons d'élevage, dont le nombre est estimé à environ 10-13 par personne.",
-  "scale_of_suffering.in_the_wild": "Cependant... dans la nature :<br /><2>Pour chaque humain</2>, il y en a deux :",
-  "scale_of_suffering.wild_birds": "10-50 oiseaux sauvages",
-  "scale_of_suffering.wild_mammals": "10-100 mammifères sauvages",
-  "scale_of_suffering.reptiles_and_amphibians": "10-10,000 reptiles et amphibiens",
-  "scale_of_suffering.fish": "1,000 - 100,000 poisson",
-  "scale_of_suffering.earthworms": "10,000 - 10⁶ vers de terre",
-  "scale_of_suffering.terrestrial_arthropods": "600,000 - 7*10⁸ arthropodes terrestres",
-  "scale_of_suffering.insects": "600,000 - 7*10⁸ insectes",
-  "scale_of_suffering.in_existence": "…en existence à tout moment.",
+  "scale_of_suffering.farmed_fish": "... sans compter les poissons d'élevage, dont le nombre est estimé à environ 10-13 par personne.",
+  "scale_of_suffering.in_the_wild": "Cependant... dans la nature :<br /><2>Pour chaque humain</2>, il y en entre :",
```

```suggestion
  "scale_of_suffering.in_the_wild": "Cependant... dans la nature :<br /><2>Pour chaque humain</2>, il y en entre :",
```

"il y en a entre" / "il y a entre" ?